pubspec.yaml

```diff
@@ -20,14 +20,11 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.2
   flutter_webrtc: ^0.2.6
-  shared_preferences:
-  shared_preferences_macos:
-  shared_preferences_web:
+  shared_preferences: ^0.5.6
```

Comment:
As explained above, I believe `shared_preferences_macos` and `shared_preferences_web` are already incorporated into shared_preferences.

> This package has been endorsed, meaning that you only need to add shared_preferences as a dependency in your pubspec.yaml. It will be automatically included in your app when you depend on package:shared_preferences.

See:
- https://pub.dev/packages/shared_preferences_macos
- https://github.com/flutter/plugins/blob/master/packages/shared_preferences/shared_preferences/pubspec.yaml

Reply:
All right!